Makie/src/compute-plots.jl

```diff
@@ -285,12 +343,34 @@ function register_positions_transformed_f32c!(attr)
                 p4d = model * p4d
                 return f32_convert(f32c, p4d[Vec(1, 2, 3)])
             end
-            return (output, Mat4f(I))
+            return (output,)
         end
     end
     return
 end
 
+function register_model_f32c!(attr)
+    map!(attr, [:model, :f32c, :space], :model_f32c) do model, f32c, space
+        trans, scale = decompose_translation_scale_matrix(model)
+
+        # is_rot_free = is_translation_scale_matrix(model)
+        if !is_data_space(space) || isnothing(f32c) || (is_identity_transform(f32c) && is_float_safe(scale, trans))
+            return Mat4f(model)
+        elseif false # is_identity_transform(f32c) && !is_float_safe(scale, trans)
```

Comment:
Why do we leave the false paths in here?

Reply:
Also there seem to be 3 other functions with the same pattern?
E.g. : https://github.com/MakieOrg/Makie.jl/blob/d2544bf6c1bb6793fe5c0078b6d7ed33a790190d/Makie/src/backend-functionality.jl#L59-L97

Shouldn't this be handled by one project function?